### PR TITLE
required email for contact form

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

**Review by jayay — COMMENTED** [review]
Is there no code change necessary for the email address check? And why did `AdminPageFormSettingsContact` get renamed?

**Comment by hungnc89** [comment]
@jayay . `AdminPageFormSettingsValidate` will handle email validation. now the contact form also needs validation so all 3 forms will use this class. I changed the name `AdminPageFormSettingsInterestOwner` to be more appropriate

File: plugin/Model/FormModelBuilder/FormModelBuilderDBForm.php [hunk 1]
```diff
@@ -272,7 +272,7 @@ public function createInputModelRecipient()
 			(InputModelDBFactoryConfigForm::INPUT_FORM_RECIPIENT, $labelRecipient);
 		$pInputModelFormRecipient->setHtmlType(InputModelOption::HTML_TYPE_TEXT);
 		$pInputModelFormRecipient->setValue($selectedRecipient);
-		$pInputModelFormRecipient->setPlaceholder(__('john.doe@example.com', 'onoffice-for-wp-websites'));
+		$pInputModelFormRecipient->setPlaceholder(__('', 'onoffice-for-wp-websites'));
```
Comment (jayay) [inline plugin/Model/FormModelBuilder/FormModelBuilderDBForm.php:275]: This entire line is a no-op and can be removed

Comment (hungnc89) [reply plugin/Model/FormModelBuilder/FormModelBuilderDBForm.php:275]: I will update it

**Comment by hungnc89** [comment]
@jayay Please check it

**Review by jayay — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**